server/apps/immich/src/api-v1/config/config.service.ts

```diff
@@ -0,0 +1,20 @@
+import { SystemConfigService } from '@app/system-config';
+import { Injectable } from '@nestjs/common';
+import { UpdateSystemConfigDto } from './dto/update-system-config';
+import { SystemConfigResponseDto } from './response-dto/system-config-response.dto';
+
+@Injectable()
+export class ConfigService {
```

Comment:
Maybe changing the class name to `ImmichConfigService` would be better in the long run.